graphjet-core/src/test/java/com/twitter/graphjet/hashing/ArrayBasedLongToInternalIntFixedLengthBiMapTest.java

```diff
@@ -80,4 +85,78 @@ public void testRandomConcurrentReadWrites() {
     InternalIdMapConcurrentTestHelper.testRandomConcurrentReadWriteThreads(
         map, -1, 600, maxNumKeys, random);
   }
+
+    @Test
+    public void testClear() {
+        DefaultStatsReceiver defaultStatsReceiver = new DefaultStatsReceiver("a");
```

Comment:
Could you use NullStatsReceiver nullStatsReceiver = new NullStatsReceiver() everywhere instead of DefaultStatsReceiver("a")? In fact, you can just use new NullStatsReceiver() whenever a statsReceiver is expected in the codes without having to declare a separate variable.